examples/sudoku_zkp/Cargo.toml

```diff
@@ -4,5 +4,5 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-sunscreen = { path = "../../sunscreen" }
-sunscreen_zkp_backend = { path = "../../sunscreen_zkp_backend" }
\ No newline at end of file
+sunscreen = { path = "../../sunscreen", features = ["debugger"]}
```

Comment:
Remove this feature. I think the debugger example should be the only one to use this.